scene/gui/color_picker.cpp

```diff
@@ -360,6 +384,16 @@ void ColorPicker::_update_controls() {
 		alpha_label->hide();
 	}
 
+	if (edit_intensity) {
+		intensity_value->show();
+		intensity_slider->show();
+		intensity_label->show();
```

Comment:
This can be changed to
```suggestion
	intensity_value->set_visible(edit_intensity);
	intensity_slider->set_visible(edit_intensity);
	intensity_label->set_visible(edit_intensity);
```